src/main/java/io/kestra/plugin/couchbase/CouchbaseConnection.java

```diff
@@ -0,0 +1,51 @@
+package io.kestra.plugin.couchbase;
+
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.ClusterOptions;
+import com.couchbase.client.java.env.ClusterEnvironment;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.runners.RunContext;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+public abstract class CouchbaseConnection extends Task implements CouchbaseConnectionInterface {
+    @NotNull
```

Comment:
Move them in the interface

Reply:
See above

Reply:
The validation annotation can be moved to the interface.